JS-103: JavaScript Functions/challenges/js/challenge4.js

```diff
@@ -0,0 +1,9 @@
+function max(num1, num2) {
+  if (num1 > num2) {
```

Comment:
@jeremyhue1999 I know this is a challenge for if/else but in practice, we can simplify this as:
```
if (num1 > num2) return true;
return false;
```